src/game/common/sparsematchfinder.h

```diff
@@ -17,18 +17,97 @@
 #include "always.h"
 #include <map>
 
-template<typename K, typename T> class SparseMatchFinder
+template<typename T, typename K> class SparseMatchFinder
```

Comment:
Can you label them descriptive names so that anyone can understand them. Suggest `Key` and `Value`

Reply:
I didn't invent those names, I just reversed their order. They were OK when this class was created, I don't see why they are a problem now...

Reply:
Renamed them anyway rather than argue :)